content/documentation/download.md

```diff
@@ -43,22 +43,44 @@ We currently do not provide stable snaps.
 
 The following requirements apply to the installation from source:
 
-* Go 1.9+
+* Go 1.10+
```

Comment:
Does this mean that I can get rid of those extras routes in the ipfsconn?

Reply:
Yes